camera-core/res/values/attrs.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <declare-styleable name="CameraView">
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <declare-styleable name="StripeCameraView">
         <!--defines the shape, aspect ratio of the viewfinder-->
-        <attr name="viewFinderType" format="enum">
+        <attr name="stripeViewFinderType" format="enum">
```

Comment:
@ccen-stripe and @awush-stripe: It looks to me like these are only intended for internal use. Is that correct?

Reply:
yes it is!, StripeCameraView is not supposed to be public